docs/src/main/sphinx/connector/phoenix.md

```diff
@@ -270,6 +270,7 @@ statements, the connector supports the following features:
 
 - {doc}`/sql/insert`
 - {doc}`/sql/delete`
+- {doc}`/sql/update`
```

Comment:
Other connectors use insert -> update -> delete order. Let's follow that for consistency.

Reply:
agreed